csharp/src/Google.Protobuf.Test/TestProtos/UnittestProto3.cs

```diff
@@ -209,8 +209,8 @@ public enum TestEnumWithDupValue {
     [pbr::OriginalName("FOO1")] Foo1 = 1,
     [pbr::OriginalName("BAR1")] Bar1 = 2,
     [pbr::OriginalName("BAZ")] Baz = 3,
-    [pbr::OriginalName("FOO2")] Foo2 = 1,
-    [pbr::OriginalName("BAR2")] Bar2 = 2,
+    [pbr::OriginalName("")] Foo2 = 1,
+    [pbr::OriginalName("")] Bar2 = 2,
   }
```

Comment:
Does Empty the OriginalNameAttribute for duplicate enum values make sense to you? So there's no need to add more attribute. We do not need the OriginalName for duplicate enum.

If so, I will go ahead change the rest code and protoc.